__test__/op-connect.test.ts

```diff
@@ -169,6 +170,54 @@ describe("Test OnePasswordConnect CRUD", () => {
         expect(vaults[0].name).toEqual(title);
         expect(vaults[1].name).toEqual(title);
     });
+
+    describe("getVaultByTitle", () => {
```

Comment:
Tiny nitpick which shouldn't affect this PR, but to have the same "pattern" with other test descriptions, what do you think of this one?
```suggestion
    describe("get vault by title", () => {
```

Reply:
I usually used exact method name in describe to quickly find a section where the tests are for this exact method.
But yeah, can change as you suggested

Reply:
We can do a refactoring of naming tests later when we will improve the overall code quality of this SDK. 😄 

But until then, let's be consistent.